server/controllers/UsersController.js

```diff
@@ -19,12 +21,18 @@ class UsersController {
     const { fullName, email, password } = req.body;
     const hashedPassword = bcrypt.hashSync(`${password}`);
     const lifeSpan = 60 * 60 * 24;
+    let token;
     return User
       .create({
         fullName,
         email,
         password: hashedPassword
       })
+      .then((user) => {
```

Comment:
Whats the need for this .then? There seems to be another starting on line 36.

Reply:
Oh... I am refactoring that now.